src/constants/localizedConstants.ts

```diff
@@ -0,0 +1,275 @@
+import * as nls from 'vscode-nls';
+
+// Localizable Strings (localized at runtime)
```

Comment:
If the string is to be localized then yes, you would want to add a string to this file and then to the xliff file. I was thinking of adding another automation step (like the tools service has) where you just add your new string to a json file and  run a gulp task to put it into the XLIFF and into the proper constants file.